authentication/keys/public.py

```diff
@@ -0,0 +1,9 @@
+PUBLIC_KEY = b"""-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAoWPv5B06KeQPru1unPs1
+k/iAME1nIEsoWJIkmaP5broZcUpztOpmRAf8+RCk5oAXQrJ87SoIs8UG4fKCduNa
+FXsKvv1zn6z+S92ScpHOYmuxO9mlL1np9NbhRdiSDwIomdpXNZ+ttF+Ll0nO87wU
+aKvTdwLhLswSLEjt2YwmKOhgpSPe2mLkk5Me1v7iAfj4ermisaV/WLu7QbelboZO
+Yp3yhOkPVKhvJU4fBkx/cnkcvLMo8dx5sM3TQonZB5oiv1w4YWz7nBjbmH3bv+9u
+AiLpr9K2ueieZbM1R3f7vhS44pNUI5LYRYA4NHEv1O+n5bE8BuvnVMG5qGP6fB8J
+uwIDAQAB
+-----END PUBLIC KEY-----"""
```

Comment:
Let's not put SSH keys directly in the code but add a CLI tool to generate this file when needed. That way we'll ensure dev and prod have different keys

Reply:
This key depends on the private key which is on the SSO, thus generated somewhere else
I don't know how to do this cleanly

Reply:
I mean, for me, it's clean enough, like, do we have another option than manually transferring the public key?

Reply:
Maybe not, but in that case just put a string that is explicitly not a key so it will fail if someone tries to use the code as it

Reply:
Sorry, what do you mean?

Reply:
In summary : don't put public or private keys in the code. If you need to make then explicitly wrong ("dummy public key" for instance)